src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int firstSide;
+    private int secondSide;
+
+    public RightTriangle(int firstSide, int secondSide, Color color) {
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
+        super.color = color;

```

Comment:
Avoid calling the superclass's fields directly; use a constructor for initialization. You should call `super(color)` here instead of `super.color = color;` to set the `color` field in the `Figure` class.